pkg/state/state_ofed.go

```diff
@@ -188,6 +294,34 @@ func (s *stateOFED) getManifestObjects(
 		}
 	}
 
+	additionalVolMounts := additionalVolumeMounts{}
+	osname := attrs[0].Attributes[nodeinfo.AttrTypeOSName]
+	// set any custom ssl key/certificate configuration provided
+	if cr.Spec.OFEDDriver.CertConfig != nil && cr.Spec.OFEDDriver.CertConfig.Name != "" {
+		destinationDir, err := getCertConfigPath(osname)
+		if err != nil {
+			return nil, fmt.Errorf("failed to get destination directory for custom repo config: %v", err)
+		}
+
+		err = s.mountAdditionalVolumesFromConfigMap(&additionalVolMounts, cr.Spec.OFEDDriver.CertConfig.Name, destinationDir)
+		if err != nil {
+			return nil, fmt.Errorf("failed to custom certificates: %v", err)
```

Comment:
> failed to custom certificates 

this is not a sentence.

maybe "failed to generate configuration for custom certificates: %v"